RecoLocalCalo/HcalRecAlgos/src/PulseShapeFitOOTPileupCorrection.cc

```diff
@@ -335,15 +335,16 @@ void PulseShapeFitOOTPileupCorrection::apply(const CaloSamples & cs,
       tsTOTen += energy - peden;
       if( ip ==4 || ip==5 ){
          tstrig += charge - ped;
+	 tstrigEn += energy - peden;
       }
    }
 
-   ts4Max_=vts4Max_[0]; //  HB and HE are identical for Run2
+   ts4Max_=vts4Max_[0]; //  HB and HE are identical for Run2, ts4Max_ in GeV for the 2017
```

Comment:
I don't quite understand the added comment ``` ts4Max_ in GeV for the 2017 ```
is this the 2017 detector configuration, or is it because this PR is submitted essentially in 2017?

Maybe just leave ```ts4Max_ in GeV ``` ?